.github/workflows/pr_review.yml

```diff
@@ -30,8 +30,8 @@ jobs:
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
         with:
           debug: true
-          review_simple_changes: false
-          review_comment_lgtm: false
+#          review_simple_changes: false
+#          review_comment_lgtm: false
           max_files: 1
           disable_release_notes: true
```

Comment:
The 'disable_release_notes' parameter seems to be set to 'true'. Is it intentional?